<ui version="4.0" >
 <class>OmgAlgorithmManager</class>
 <widget class="QDialog" name="OmgAlgorithmManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Algorithm Manager</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Algorithm profiles</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="3" >
        <widget class="QTableWidget" name="tblAlgorithms" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="showGrid" >
          <bool>false</bool>
         </property>
         <property name="gridStyle" >
          <enum>Qt::NoPen</enum>
         </property>
         <row>
          <property name="text" >
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text" >
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Algorithm</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>81</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="pbnClone" >
         <property name="minimumSize" >
          <size>
           <width>28</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="text" >
          <string>Clone</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QPushButton" name="pbnDelete" >
         <property name="minimumSize" >
          <size>
           <width>28</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
         <property name="text" >
          <string>Delete</string>
         </property>
         <property name="iconSize" >
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="grpDetails" >
      <property name="title" >
       <string>Details</string>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QTabWidget" name="tabMetadata" >
         <property name="tabPosition" >
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab" >
          <attribute name="title" >
           <string>Profile</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="3" column="0" >
            <widget class="QLabel" name="label_3" >
             <property name="text" >
              <string>Overview</string>
             </property>
             <property name="buddy" >
              <cstring>teOverview</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLineEdit" name="leProfileName" />
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string>Profile name</string>
             </property>
             <property name="buddy" >
              <cstring>leProfileName</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="0" >
            <widget class="QTextEdit" name="teOverview" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="lblAlgorithm" >
             <property name="text" >
              <string>Algorithm</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabParameters" >
          <attribute name="title" >
           <string>Parameters</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="1" column="0" >
            <widget class="QLabel" name="lblParameterCount" >
             <property name="text" >
              <string>Parameter count</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>281</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="3" >
            <widget class="QFrame" name="frameParameters" >
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QPushButton" name="pbnDefaults" >
             <property name="text" >
              <string>Defaults</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabDetails" >
          <attribute name="title" >
           <string>Description</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="lblVersionLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Version</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="lblAuthorLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Author</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="lblImplementedByLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Code author</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2" >
            <widget class="QLabel" name="lblCodeAuthor" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>[code author goes here]</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2" >
            <widget class="QLabel" name="lblVersion" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>[version goes here]</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3" >
            <widget class="QTextEdit" name="teDescription" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="3" >
            <widget class="QLabel" name="lblDescriptionLabel" >
             <property name="text" >
              <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:10pt; font-weight:600;">&lt;span style=" font-weight:400;">Description&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
             </property>
             <property name="buddy" >
              <cstring>teDescription</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="2" >
            <widget class="QCheckBox" name="cbxAbsence" >
             <property name="text" >
              <string>Uses absence data</string>
             </property>
             <property name="checkable" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2" >
            <widget class="QCheckBox" name="cbxCategorical" >
             <property name="text" >
              <string>Uses categorical data</string>
             </property>
             <property name="checkable" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2" >
            <widget class="QLabel" name="lblAuthor" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>[author goes here]</string>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabBibliography" >
          <attribute name="title" >
           <string>Bibliography</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QTextEdit" name="teBibliography" >
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="lblProfileIcon" >
         <property name="maximumSize" >
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="pixmap" >
          <pixmap resource="../resources/resources.qrc" >:/algorithmTypeSystem.png</pixmap>
         </property>
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="lblProfileNotes" >
         <property name="text" >
          <string>Non-editable system profile.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>681</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="pbnApply" >
     <property name="text" >
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="pbnClose" >
     <property name="text" >
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tblAlgorithms</tabstop>
  <tabstop>pbnClone</tabstop>
  <tabstop>pbnDelete</tabstop>
  <tabstop>tabMetadata</tabstop>
  <tabstop>leProfileName</tabstop>
  <tabstop>teOverview</tabstop>
  <tabstop>pbnApply</tabstop>
  <tabstop>teDescription</tabstop>
  <tabstop>teBibliography</tabstop>
  <tabstop>pbnDefaults</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbnClose</sender>
   <signal>clicked()</signal>
   <receiver>OmgAlgorithmManager</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>701</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel" >
     <x>753</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
